server/metrics/metronome.go

```diff
@@ -15,44 +15,50 @@
 package metrics
 
 import (
+	"github.com/tigrisdata/tigris/server/config"
 	"strconv"
 
 	"github.com/uber-go/tally"
 )
 
 var (
-	MetronomeCreateAccount tally.Scope
-	MetronomeAddPlan       tally.Scope
-	MetronomeIngest        tally.Scope
-	MetronomeListInvoices  tally.Scope
-	MetronomeGetInvoice    tally.Scope
-	MetronomeGetUsage      tally.Scope
-	MetronomeGetCustomer   tally.Scope
+	MetronomeRequestOk         tally.Scope
+	MetronomeRequestError      tally.Scope
+	MetronomeResponseTime      tally.Scope
+	MetronomeErrorResponseTime tally.Scope
+	MetronomeIngest            tally.Scope
 )
 
 func initializeMetronomeScopes() {
-	MetronomeCreateAccount = MetronomeMetrics.SubScope("create_account")
-	MetronomeAddPlan = MetronomeMetrics.SubScope("add_plan")
+	MetronomeRequestOk = MetronomeMetrics.SubScope("count")
+	MetronomeRequestError = MetronomeMetrics.SubScope("count")
+	MetronomeResponseTime = MetronomeMetrics.SubScope("response")
+	MetronomeErrorResponseTime = MetronomeMetrics.SubScope("error_response")
 	MetronomeIngest = MetronomeMetrics.SubScope("ingest")
-	MetronomeListInvoices = MetronomeMetrics.SubScope("list_invoices")
-	MetronomeGetInvoice = MetronomeMetrics.SubScope("get_invoice")
-	MetronomeGetUsage = MetronomeMetrics.SubScope("get_usage")
-	MetronomeGetCustomer = MetronomeMetrics.SubScope("get_customer")
 }
 
-func GetResponseCodeTags(code int) map[string]string {
+func getMetronomeTagKeys() []string {
+	return []string{
+		"env",
```

Comment:
do we need env?

Reply:
Yes, and it it's not here, it will be stripped by standardizeTags